BnbnavNetClient/Views/MapView.axaml.cs

```diff
@@ -19,6 +20,8 @@ public partial class MapView : UserControl
 {
     bool _pointerPressing;
     Point _pointerPrevPosition;
+    Point _viewVelocity;
```

Comment:
use semantically more correct ``Vector`` datatype instead, intiialize to ``Vector.Zero`` here